.travis.yml

```diff
@@ -5,12 +5,17 @@ php:
   - 5.6
   - 7
 
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 before_install:
   - composer self-update
   - composer validate
+  - composer config github-oauth.github.com $GITHUB_TOKEN
```

Comment:
@alieander @thommahoney 

This is useful to prevent hitting the GitHub API rate limit, but requires that you provide a GitHub token at https://travis-ci.org/fastly/fastly-php/settings, ideally without displaying the value!

![screen shot 2016-06-29 at 11 12 48](https://cloud.githubusercontent.com/assets/605483/16446940/754154a8-3dea-11e6-9e48-bb90961a8f44.png)